Source/WebKit/Shared/API/glib/WebKitContextMenuItem.cpp

```diff
@@ -357,6 +360,32 @@ WebKitContextMenuAction webkit_context_menu_item_get_stock_action(WebKitContextM
 #endif // ENABLE(CONTEXT_MENUS)
 }
 
+#if PLATFORM(WPE)
+/**
+ * webkit_context_menu_item_get_title:
+ * @item: a #WebKitContextMenuItem
+ *
+ * Gets the title of @item.
+ *
+ * Returns: the title of @item, or %NULL if @item is a separator.
```

Comment:
This is missing an introspection annotation to indicate that the returned string is owned by `WebKitContextMenu`:

```suggestion
 * Returns: (transfer none): the title of @item, or %NULL if @item is a separator.
```